opennms-base-assembly/src/main/filtered/etc/otrs.properties

```diff
@@ -52,6 +52,9 @@
 # Typical OTRS v 3.1.x and above endpoint
 #otrs.endpoint=http://localhost/otrs/nph-genericinterface.pl/Webservice/GenericTicketConnector
 
+# OTRS v 3.1.x and above require a soapAction HTTP header
+otrs.soapaction=http://www.otrs.org/TicketConnector/
```

Comment:
Is this something which should be commented in by default like the `ours.endpoint` property?